master_changes.md

```diff
@@ -5,6 +5,10 @@ Prefixes used to help generate release notes, changes, and blog posts:
 * ◈ New option/command/subcommand
 * [BUG] for bug fixes
 * [NEW] for new features (not a command itself)
+* 🕮i api update
```

Comment:
What's the emoji bit for, or is my browser being strange?

I'm not utterly convinced that this is a good idea, but let's see how it goes!

Reply:
héhé, it is just a book for _library_ , maybe we should have the more intelligible `[API]`. The idea is to point API breakage/updates that can affect library users